central/sensor/enhancement/broker.go

```diff
@@ -51,8 +53,10 @@ func (b *Broker) NotifyDeploymentReceived(msg *central.DeploymentEnhancementResp
 		log.Warnf("Received response to an unknown Deployment Enrichment Request ID %s", reqID)
 		return
 	}
-	log.Debugf("Received answer for Deployment Enrichment Request ID %s", reqID)
+	elapsed := time.Now().UnixMilli() - sig.requestTime.UnixMilli()
```

Comment:
We could use here https://pkg.go.dev/time#Since or `time.Now().Sub(the other time)` as they are robust against "wall clock is changed during the operation being timed"

Reply:
Done

Reply:
Lol, we now have a linter for that 😂 

```
central/sensor/enhancement/broker.go:56:13: S1012: should use `time.Since` instead of `time.Now().Sub` (gosimple)
```